libuuu/cmd.cpp

```diff
@@ -415,6 +425,73 @@ int CmdDelay::run(CmdCtx *)
 	return 0;
 }
 
+int CmdConfig::parser(char * /*p*/)
+{
+	try 
+	{
+	    size_t pos = 0;
+	    string param = get_next_param(m_cmd, pos);
+
+	    if (param.find(':') != string::npos)
+		    param = get_next_param(m_cmd, pos);
+
+	    if (str_to_upper(param) != "CONFIG")
+		{
+		    string err = "CmdConfig::parser Uknown Commnd:";
+		    err += param;
+		    set_last_err_string(err);
+		    return -1;
+		}
+
+	    string args = get_next_param(m_cmd, pos);
+	    if (str_to_upper(args) == "USB::HID::MAX_PACKET_SIZE")
+		{
+		    string size = get_next_param(m_cmd, pos);
+		    Conf::GetConf().m_USB.m_HIDMaxTransfer = str_to_uint(size);
+		    cout << "changed USB::HID::MAX_PACKET_SIZE to " << Conf::GetConf().m_USB.m_HIDMaxTransfer << endl;
+		}
```

Comment:
Avoid print at libuuu, which will block UI deplay. 
Please use notification function to tell uplayer application. Show or hide this information decide by application.